integration_test/util/restartableSyncService.js

```diff
@@ -15,7 +15,7 @@ class RestartableSyncService extends DataSyncService {
     this.app.server.stop()
     await this.initialize()
     await this.start()
-    await new Promise(resolve => setTimeout(resolve, 1000))
+    await new Promise(resolve => setTimeout(resolve, 200))
```

Comment:
The same of above _IMHO in order to follow the best practices and to avoid magic numbers the 200 here should be a const with a name which clarifies its propose._ 

Note that it has been used in more than one place too.